docs/developer_guide.md

```diff
@@ -0,0 +1,75 @@
+# Developer guide
+
+## Leapp tool
+### Installing on RHEL 7
+
+**Register & subscribe**
+```
+subscription-manager register
+subscription-manager list --available --match-installed
+subscription-manager attach --pool POOL_ID
+```
+
+**Update to the latest `7.x` version**
+```
+yum clean all
+yum update
+reboot
+
+cat /etc/os-release
+# => RHEL is on latest version
+```
+
+**Install Leapp tool**
+```
+subscription-manager repos --enable rhel-7-server-rpms
+subscription-manager repos --enable rhel-7-server-extras-rpms
+
+yum install -y leapp
+```
+### Installing on CentOS 7
+
+!!! There is an error when running `leapp preupgrade` command on `CentOS 7`, better try it on `RHEL 7` !!!
+
+**Update to the latest `7.x` version**
+```
+yum clean all
+yum update
+reboot
+
+cat /etc/os-release
+```
+
+**Install Leapp**
+```
+yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/oamg/leapp/repo/epel-7/group_oamg-leapp-epel-7.repo
+
+yum install yum-utils leapp
+```
+
+## Leapp commands
+
+**Pre-upgrade check**
+```
+leapp preupgrade
```

Comment:
can you add the same warning here that currently there's an error with it and better try with RHEL?

Reply:
I will remove the whole section with CentOS. We should test it only on RHEL systems anyway